### DISCUSSION TITLE
Deployed my own API server but unable to download the dataset

### DISCUSSION THREAD

**Discussion by Bhanup96** [open-meteo/open-meteo#558] [Q&A]
docker run -it --rm -v ${PWD}/data_weather:/app/data ghcr.io/open-meteo/open-meteo download-ecmwf --run 00 --only-variables temperature_2m
[ INFO ] Downloading domain ECMWF run '2023-12-20T00:00'
[ INFO ] Downloading height and elevation data
[ INFO ] Downloading file https://data.ecmwf.int/forecasts/20231220/00z/0p4-beta/oper/20231220000000-0h-oper-fc.index
[ INFO ] Download failed, retry every 5 seconds, (0 minutes elapsed, curl error 'downloadFailed(code: 404 Not Found)'
^C[ INFO ] Download failed, retry every 5 seconds, (1 minutes elapsed, curl error 'downloadFailed(code: 404 Not Found)'

and I also want to know how to see the historical data and auto refresh the data daily

**Comment by patrick-zippenfenig** [comment]
Hi, right now the entire data download system is being rewritten. Soon you will be able to directly synchronise the Open-Meteo database without having to download raw data from various weather services (should be ready in 2-3 weeks)

The issue above is most likely expected, because the 0z run ECMWF had not been available at this time. ECMWF has a rather large delay. A cronjob definition is available here: https://github.com/open-meteo/open-meteo/blob/main/Sources/App/Commands/CronjobCommand.swift

**Reply by Bhanup96** [reply]
Can you update me when it is directly synchronise the open meter database and the sets to setup my own server

**Reply by patrick-zippenfenig** [reply]
The new open-data distribution is now available and getting-started instruction have been updated https://github.com/open-meteo/open-meteo/releases/tag/1.0.0
